examples/simple/main.tf

```diff
@@ -0,0 +1,44 @@
+#
+# ECS Service Module
+#
+
+module "app_ecs_service" {
+  source = "../../"
```

Comment:
should this not be from the registry https://registry.terraform.io/modules/trussworks/ecs-service/aws/2.0.2
To be honest, I do not understand why in some examples there's a registry reference and some are like this. Is it that you use the "../../" when referencing the module this is an example for? If so, I think a better example is to show how it might be used by others, in which case the registry source is more representative.

Reply:
Using `../../` is meant to test local changes to the module. This will let me know if I have introduced breaking changes before releasing a new version to the registry.